src/BaseballGame.js

```diff
@@ -0,0 +1,99 @@
+const { Console } = require('@woowacourse/mission-utils');
+const Messages = require('./constants/Messages');
```

Comment:
개인적으론 상수는 모두 대문자로 쓰는걸 좋아한답니다!
```suggestion
const MESSAGES = require('./constants/Messages');
```
이런 것도 취향이 갈리다니 신기하지 않나요?ㅎㅎ

Reply:
저도 상수에는 대문자를 쓰는 걸 선호하지만 상수를 담는 바구니는 변수라고 생각이 들어 요렇게 작성하였습니다!
협업 진행할 때는 이야기해서 맞춰보아요~!